debian/patches/debian-changes

```diff
@@ -0,0 +1,20 @@
+Description: Debian-specific changes mixed together.
+ This patch contains all the Debian-specific changes mixed together. To
+ review them separately, please inspect the Git history (see the
+ Vcs-Git field in debian/control); the patches against upstream are all
+ on the patched branch.
+ .
+ For more information, please see debian/README.source.
+Origin: Debian
+
+--- syslog-ng-3.8.0alpha0.orig/autogen.sh
++++ syslog-ng-3.8.0alpha0/autogen.sh
+@@ -58,7 +58,7 @@ autogen_submodules()
+ 			./autogen.sh
+ 			unset NOCONFIGURE
+ 		elif [ -f configure.in ] || [ -f configure.ac ]; then
+-			autoreconf -i
++			autoreconf --force -i
+ 		else
+ 			echo "Don't know how to bootstrap submodule '$submod'" >&2
```

Comment:
I think this extra addition is not required... I'll remove it (I was needed for the force option because of an OBS issue). I'll remove this.